src/pages/Cart/Cart.js

```diff
@@ -1,104 +1,146 @@
-import { check } from 'prettier';
 import React, { useState, useEffect } from 'react';
-import { Navigate, useNavigate } from 'react-router-dom';
-import CartList from '../CartList/CartList';
+import { useNavigate } from 'react-router-dom';
 import './Cart.scss';
+import CartEmpty from './components/CartEmpty';
+import CartItem from './components/CartItem';
 
-function Cart() {
+const Cart = () => {
+  const [cartList, setCartList] = useState([]);
+  const [sumPrice, setSumPrice] = useState(0);
+  const navigate = useNavigate();
+  const accessToken = localStorage.getItem('token');
```

Comment:
accessToken 이 없을 경우를 생각해서 널리시 컬랩싱 해주면 더 좋을 것 같습니다!!
```suggestion
  const accessToken = localStorage.getItem('token') ?? '';
```